Other/Sigil/TimeStreamRavagersSigil.cs

```diff
@@ -0,0 +1,39 @@
+//cs_include Scripts/CoreBots.cs
+
+using Skua.Core.Interfaces;
+using Skua.Core.Options;
+
+public class TimeStreamRavagersSigil
```

Comment:
it is not made but for being a decent bot try to make farm all the other drops from the boss/map instead of just 1 weapon

Reply:
i think the others already have merged bot made and is only missing this 1.
![image](https://user-images.githubusercontent.com/58910880/203886652-73f1176c-6850-47de-8093-12b4fd187e8a.png)

Reply:
i think i'll remake this bot, to farm the sets the boss drops. ty for feedback

Reply:
study material: Templates\QuestRewardTemplate.cs

Reply:
@wtffidy the timestream sigil isn't a reward but a drop

Reply:
o.